app/controllers/application_controller.rb

```diff
@@ -1,7 +1,12 @@
 class ApplicationController < ActionController::Base
```

Comment:
- [ ] [OPTIONAL] Your app is accessible without the need for authentication, it would be great if you prevent users from accessing the app without login or creating new account first